### PR TITLE
Case-insensitive regexes for checking image extensions #270

### PR DESCRIPTION
**Pull request by ryki2658**
…image extensions

### CODE REVIEW

**Comment by emackey** [comment]
Fixes #270

File: lib/loadGltfUris.js [hunk 1]
```diff
@@ -169,7 +168,7 @@ var jimpSupportedExtensions = ['.png', '.jpg', '.jpeg', '.bmp'];
 function generateJimpImage(object) {
     var pipelineExtras = object.extras._pipeline;
     var ext = pipelineExtras.extension;
-    if (jimpSupportedExtensions.indexOf(ext) === -1) {
+    if (supportedExtensions.test(ext)) {
```
Comment (lilleyse) [inline lib/loadGltfUris.js]: This should be `!supportedExtensions.test(ext)`

**Comment by mramato** [comment]
Thanks @ryki2658, the tests are failing because you reversed the if check.  Previously it was checking if the extension was not in the list, now it's checking if it is in the list.  You need to add a `!` in front of the check and then this will be ready.

File: lib/loadGltfUris.js (hunk 1 above)
Comment (lilleyse) [inline lib/loadGltfUris.js]: Looks like me and @mramato replied at about the same time.

Comment (ryki2658) [inline lib/loadGltfUris.js]: Silly mistake sorry. I appreciate the patience with me muddling my way through this first attempt at GitHub and my first attempt at contributing to a "real" project. I hope I haven't been to much of a disturbance in the operation of things for everyone. Thank you for allowing me to participate.

Comment (mramato) [inline lib/loadGltfUris.js]: Don't be so hard on yourself.  We are were all first time committers at some point.  We are happy to help and appreciate your help.

**Comment by mramato** [comment]
Thanks again, @ryki2658.

**Comment by ryki2658** [comment]
No problem happy to be involved!

**Pull request merged**